### ISSUE TITLE
Question + Minor comment

### ISSUE THREAD

**Issue by ptoche** [Groupe-ElementR/SpatialPosition#19]
The [vignette](https://cran.r-project.org/web/packages/SpatialPosition/vignettes/StewartExample.html) is fantastic! 

**Question**: Do you have a version based on `mapsf`? As I'm new to this I wanted to learn straight from the successor to `cartography`.

Minor comment: GDP is an income concept, not a wealth concept. So the title should be "Income Inequality in Europe" or more precisely "GDP per capita Inequality in Europe".  European wealth is in fact much higher. See Piketty and Zucman on this.

**Comment by rCarto** [comment]
Hi, 
My advise is to use `potential` + `mapsf` instead of `SpatialPosition` + `cartography`. 
I have just updated the documentation of potential to use `mapsf`. 
See https://riatelab.github.io/potential/ for the full documentation and https://riatelab.github.io/potential/articles/exemple.html for the use case with `mapsf`.

I changed the map titles according to your suggestion, "wealth" was used to simplify the title but you're right, it is eventually misleading.

**Comment by ptoche** [comment]
Fantastic! Great work, thanks!

**Comment by ptoche** [comment]
As an afterthought: It's not clear to me how to use `mf_export()`. I tried

    mf_export(x = world, filename = "maps-inequality-europe-basic.png", width = 600) 
    dev.off()

but the exported image is blank. 

I also tried variations on

    filename <- tempfile(fileext = ".png")
    mf_export(world, filename = filename)
    mf_map(world, add = TRUE)
    dev.off()

 based on the example in the help page, but that produced in the current directory.

Thanks.

References:

https://riatelab.github.io/potential/articles/exemple.html
https://riatelab.github.io/mapsf/reference/mf_export.html
